transformers/websockets/server.js

```diff
@@ -51,9 +51,6 @@ module.exports = function server() {
 
       socket.on('message', spark.emits('incoming::data'));
       socket.on('error', spark.emits('incoming::error'));
-      socket.on('ping', spark.emits('incoming::ping', (next) => {
```

Comment:
I've removed this as there is no way to send ping frames from the browser, and we are not using the WebSocket pings in our client.